src/main/java/com/beecoders/ras/model/request/AddDishRequest.java

```diff
@@ -0,0 +1,28 @@
+package com.beecoders.ras.model.request;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Builder
+@EqualsAndHashCode
+public class AddDishRequest {
+    private String name;
+    private String description;
+    private Integer weight;
+    private Double price;
+    private List<String> specifics;
+    @JsonProperty("ingredientsId")
+    private List<Long> ingredientsId;
+    @JsonProperty("categoryId")
+    private Long categoryId;
```

Comment:
It would be good to add `JsonProperty` for all attributes to avoid any related exceptions in future

Reply:
Added